### PR TITLE
Newsletter 233 translate in french

### PR DESCRIPTION
**Pull request by Copinmalin**

### CODE REVIEW

File: _posts/fr/newsletters/2023-01-11-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+    et qu'elle décide de faire un paiement régulier sur la chaîne, il lui
+    suffit que le portefeuille de Bob cosigne la dépense. Dans le pire des cas,
+    si Bob ne coopère pas, Alice peut simplement attendre quelques semaines
+    pour dépenser son argent sans sa participation.
+
+    En plus de permettre aux portefeuilles hors ligne de recevoir des fonds pour LN,
+    les auteurs décrivent comment l'idée pourrait bien se combiner avec les
+    [paiements asynchrones][topic async payments] pour permettre aux LSP (fournisseur
+    de service de liquidité) de préparer les opérations de rééquilibrage
+    des canaux à l'avance pour le moment où un client hors ligne revient en ligne,
```
Comment (Jluc-bitcoinfr) [inline _posts/fr/newsletters/2023-01-11-newsletter.md:88]: des canaux à l'avance avant qu'un client hors ligne ne revienne en ligne,

File: _posts/fr/newsletters/2023-01-11-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+    permettant à ces opérations de rééquilibrage de se produire sans aucun retard
+    du point de vue de l'utilisateur. Par exemple, si Carole envoie un paiement
```
Comment (Jluc-bitcoinfr) [inline _posts/fr/newsletters/2023-01-11-newsletter.md:90]: pour l'utilisateur. Par exemple, si Carole envoie un paiement

**Review by bitschmidty — COMMENTED** [review]
@Copinmalin I rebased on master and then cherry-picked the commits related to this newsletter (green in screenshot).

This PR also had content from the 180 translation and 232 translation which caused merge conflicts with how you had this PR setup. To prevent, make sure new branches are based on an updated origin/master before adding a new translation.
Screen Shot 2023-01-18 at 9 21 53 AM

**Comment by Copinmalin** [comment]
Hello Mike,

Now I'm a little bit confuse !

I need your help

How to rebase my branch now ?

-------- Courriel original --------

  		OBJET:
  		Re: [bitcoinops/bitcoinops.github.io] Newsletter 233 translate in 
french (PR #985)

  		DATE:
  		18.01.2023 16:25

  		DE:
  		Mike Schmidt ***@***.***>

  		À:
  		"bitcoinops/bitcoinops.github.io" 
***@***.***>

@bitschmidty commented on this pull request.

@Copinmalin [1] I rebased on master and then cherry-picked the commits 
related to this newsletter (green in screenshot).

This PR also had content from the 180 translation and 232 translation 
which caused merge conflicts with how you had this PR setup. To prevent, 
make sure new branches are based on an updated origin/master before 
adding a new translation.
  [2]

--
Reply to this email directly, view it on GitHub [3], or unsubscribe [4].
You are receiving this because you were mentioned.Message ID: 
***@***.***>

Links:
------
[1] https://github.com/Copinmalin
[2] 
https://user-images.githubusercontent.com/1615772/213212330-e8282738-d45f-4162-8689-3b0083d82741.png
[3] 
https://github.com/bitcoinops/bitcoinops.github.io/pull/985#pullrequestreview-1253705394
[4]

**Comment by bitschmidty** [comment]
@Copinmalin you should be able to checkout origin / master locally after doing a git fetch.

If there are conflicts you can do a reset --hard to origin master

**Pull request merged**